### ISSUE TITLE
`brew test python` *requires* the tcl-tk *option*

### ISSUE THREAD

**Issue by gsteemso** [mistydemeo/tigerbrew#1148]
What it says.  You must install an optional feature to run the Python 2.7 formula’s test method.  I would submit a patch to conditionalize it, but my `git` install is currently broken (sorry).

Also, I have found that the `tkinter` module which the test tries to load may not support Python 2 any more, or something of the sort. (I'm still trying to work that part out in detail; there may be more than one potential cause of the error message I get.)

There is one positive thing – because I've got HOMEBREW_DEBUG set in my environment, which gives me an option to ignore the error, I can say that all other parts of the test routine appear to work properly.

**Comment by sevan** [comment]
Local issue?
Test only runs `system bin/"pip", "list", "--format=columns"` at the last phase.

**Comment by gsteemso** [comment]
Yes.  Freshly installed python installs pip 20.3.4, which warns that python 2 support will be removed in pip 21.
